### PR TITLE
Week 09 day 03 mod 01

### PR DESCRIPTION
**Pull request by eerwitt**
Most of the content is on other sites. Tried to organize it in terms of the steps we'll take.

I'm thinking it might be nice to do a small exercise to show how search indexing works. I'm hoping at least two people may join for it otherwise it'd be kinda boring.

@cameronjacoby do you think it'd be OK to ask people who don't want to be involved with the lesson work from someplace outside the class while we do the exercise?

Exercise idea:
Each person comes up with a unique word (maybe their name?) . Then we start with a simple where claus to get all of a single word but quickly branch out to more advanced queries which would require stemming.

### CODE REVIEW

File: how-tos/rails/elasticsearch/readme.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+<img src="https://cloud.githubusercontent.com/assets/1329385/12210263/7e707438-b60d-11e5-86a4-da08c62efd29.gif">
+
+## What is Elasticsearch (ES)?
+
+> Elasticsearch is a search server based on Lucene. It provides a distributed, multitenant-capable full-text search engine with an HTTP web interface and schema-free JSON documents. Elasticsearch is developed in Java and is released as open source under the terms of the Apache License. Elasticsearch is the second most popular enterprise search engine after Apache Solr, also based on Lucene <a href="https://en.wikipedia.org/wiki/Elasticsearch" target="_blank">Wiki</a>
+
+We'll go over this description in class and focus on terms which may not be clear.
+
+<details>
+  <summary>Question to ask yourself, what is the difference between Elasticsearch and PostgreSQL?</summary>
```
Comment (cameronjacoby) [inline how-tos/rails/elasticsearch/readme.md]: Did you mean PostgreSQL here? Or MongoDB?

**Comment by cameronjacoby** [comment]
@eerwitt Just one comment above, but otherwise looks good!

Yes, I think it's totally fine to have students work somewhere outside the classroom if they're not coming to the lessons next week. Good idea!

And I think your plan for the exercise sounds great and will really demonstrate the concepts!

File: how-tos/rails/elasticsearch/readme.md (hunk 1 above)
Comment (eerwitt) [inline how-tos/rails/elasticsearch/readme.md]: I will make this more clear, the link talks about the differences with RDMS in particular PostgreSQL but starts out about Mongo vs ES :/

Comment (cameronjacoby) [inline how-tos/rails/elasticsearch/readme.md]: Ah, that makes sense. I didn't really read the link, just the title.

**Comment by eerwitt** [comment]
@cameronjacoby updated

**Comment by cameronjacoby** [comment]
@eerwitt Did you git push? The last commit was me.

**Comment by eerwitt** [comment]
@cameronjacoby nope, but did now :)

**Pull request merged**